FirebaseDatabaseSwift/Tests/third_party/EncoderTests.swift

```diff
@@ -0,0 +1,1874 @@
+// This file is derived from swift/test/stdlib/TestJSONEncoder.swift
```

Comment:
For future readers: diffed the original copy and this one and there were no major changes.